src/exceptions/PS-or-PDF-font-exception-20170817.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <exception licenseId="PS-or-PDF-font-exception-20170817" name="PS/PDF font exception (2017-08-17)" listVersionAdded="3.2">
```

Comment:
If we're hyphenating the date in the name, do we want to hyphenate the date in the ID as well?  I'd hyphenate both for consistency, but don't feel strongly about it if you have your own preferences.

Reply:
I think date is fine as is - the short identifier is already kind of long and the other ids we used dates in (W3C) don't have hyphens

Reply:
The difference was to match previous IDs and titles, most of which had the ID as "YYYYMMDD" and the title as "(YYYY-MM-DD)" or "(YYYY)"